src/components/organisms/Header.jsx

```diff
@@ -0,0 +1,110 @@
+import React, { useEffect, useState } from 'react';
+import { NavLink } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faBook, faBars } from '@fortawesome/free-solid-svg-icons';
+import UserAuthBtn from '../molecules/UserAuthBtn';
+
+function Header() {
+  const [collapse, setCollapse] = useState();
+  const [toggleClass, setToggleClass] = useState('');
+  const [toggleAria, setToggleAria] = useState('');
+  const [navClass, setNavClass] = useState('');
+
+  const updateToggleClass = (isActive) => {
+    if (isActive) {
+      setToggleClass('navbar-toggler collapsed');
+      setToggleAria('false');
+      setNavClass('navbar-collapse collapse');
```

Comment:
Same problem with `navbar-collapse collapse` as seen in `setToggleClass`. Move classes to appropriate location.